src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -36,10 +36,15 @@
  */
 final class ClassAttributesSeparationFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface, WhitespacesAwareFixerInterface
 {
+    const SPACING_ONE = 'one';
```

Comment:
Let's have alphabetical order of these constants (especially as they are used in alphabetical order).